tests/e2e/test_running_cluster_monitoring_with_persistent_storage.py

```diff
@@ -26,20 +26,18 @@ def test_fixture(request):
     """
     Setup and teardown
     """
-    self = request.node.cls
-
     def finalizer():
-        teardown(self)
+        teardown()
```

Comment:
Since `teardown` doesn't take any arguments we can add it as a finalizer directly.
```
request.addfinalizer(teardown)
```